### PR TITLE
Moved train_classifier file to main/

### PR DESCRIPTION
**Pull request by sophieball**
Moved the file and updated WORKSPACE and BUILD files

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -1,18 +1,6 @@
 #!/bin/bash
-bazel build //src:train_classifier_g
-
-# check if spacy en is linked
-SPACY_PATH=bazel-bin/src/train_classifier_g.runfiles/deps_pypi__spacy_2_2_4/spacy/data/
```
Comment (CaptainEmerson) [inline setup.sh:5]: ooh, is this not necessary any more? nice! if that's the case, I would say you can drop setup.sh -- it's clear enough how to build the project

File: src/requirements.txt [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
 textblob
+thinc
+threadpoolctl
+wasabi
 wordfreq
-requests
-https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.2.0/en_core_web_sm-2.2.0.tar.gz#egg=en_core_web_sm==2.2.0
+zipp
 https://github.com/explosion/spacy-models/releases/download/en_core_web_md-2.2.0/en_core_web_md-2.2.0.tar.gz#egg=en_core_web_md==2.2.0
+https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.2.0/en_core_web_sm-2.2.0.tar.gz#egg=en_core_web_sm==2.2.0
```
Comment (CaptainEmerson) [inline src/requirements.txt:32]: still confusing to me why I'm seeing changes that were present in prior PR. maybe your branch is stale? can you updated it to head?

File: main/BUILD [hunk 3]
```diff
@@ -0,0 +1,15 @@
+load("@rules_python//python:defs.bzl", "py_binary")
```
Comment (CaptainEmerson) [inline main/BUILD:1]: generally projects only have one build file at the top level directory

Comment (sophieball) [reply main/BUILD:1]: https://github.com/bazelbuild/rules_python/issues/325
The example here shows that I need a BUILD in each directory (src and main). I also tried to use one BUILD file at the top level and refer to files with relative path, but it told me to " Add a BUILD file to a directory to mark it as a package.
 - /usr/local/google/home/hsqq/toxicity/sophie/toxicity-detector/main and referenced by '//:train_classifier_g'"

Comment (CaptainEmerson) [reply main/BUILD:1]: Ok! I stand corrected :)

File: src/requirements.txt (hunk 2 above)
Comment (sophieball) [reply src/requirements.txt]: Outdated.

File: setup.sh (hunk 1 above)
Comment (sophieball) [reply setup.sh:5]: deleted.

**Pull request merged**